thesis-de/content-de.tex

```diff
@@ -361,11 +361,145 @@ \subsubsection{Durchführung der Evaluierung}
 \nl%
 
 \subsection{Frontends}
+\label{subsec:frontends}
+Bevor das Projekt beginnen konnten, musste zusammen mit dem Auftraggeber und
+dem Projektteam, eine Liste an möglichen Frontends erarbeitet werden.
+Zusätzlich zu dieser Liste, wurden verschiedene Anforderungen definiert, welche
+im späteren Verlauf der Planung für die Evaluierung der einzelnen Frontends
+verwendet wird. Diese Liste besteht zum Teil aus Software Lösungen, welche im
+eigenen Unternehmen, bereits erfolgreich eingesetzt worden sind und zum anderen
+von Vorschläge des Auftraggebers. Während der Planung des Frontends, kamen noch
+weitere Frontends zu der Liste hinzu, welche in die Evaluierung mit aufgenommen
+worden sind.
+
+Dies benötigte keine Zustimmung des Auftraggebers, sondern konnte mit dem
+Projektteam besprochen werden.
+
+Folgende Frontends wurden während der Planung, bereits vorab mit dem
+Auftraggeber und dem Porjektteam erarbeitet:
 
 \begin{outline}
   \1 grafana
   \1 graphite
   \1 zabbix-frontend
+  \1 datadog
+  \1 gridster-D3
+  \1 netdata
+\end{outline}
+\mr%
+
+\subsection{Vorbereitung der Evaluierung}
+\label{subsubsec:vorbereiten_der_evaluierung_frontend}
+Damit die Evaluierung der einzelnen Frontends durchgeführt werden konnten,
+mussten zuvor Kriterien zusammen mit dem Porjektteam definiert werden, welche
+die Frontends erfüllen müssen. (Diese sollen später für eine bessere
+Übersichtlichkeit dienen, welche Anforderungen das Frontend haben muss). Bei 
+den Kriterien, musste darauf geachtet werden, dass diese zu Projektübergabe
```

Comment:
Nicht zur Projektübergabe, sondern bei der Beurteilung der Systeme in der Evaluierung.
Wir können ja nicht am Ende des Projektes mit leeren Händen kommen und sagen während der Evaluierung traf keine Anforderung ins grüne, sodass wir nun kein Frontend besitzen. 

Beispiel:
, dass diese während der Beurteilung der Anforderungen eine Machbarkeit und Realistik besitze,